src/styles/blocks/_catalog.scss

```diff
@@ -0,0 +1,46 @@
+.catalog {
+  display: grid;
+  justify-content: center;
+  gap: $card-gap;
+
+  &__card {
+    display: grid;
+    justify-content: center;
+    align-items: center;
+    @include card($card-height, $card-width);
+    border: 1px solid $color-border;
+    border-radius: 5px;
+    color: $color-primary;
+    transition: background-color $duration, color $duration;
+
+    &:hover {
+      cursor: pointer;
+      background-color: $color-primary;
+      color: $color-secondary;
+    }
+  }
+
+  @media (min-width: 1024px) {
+    & {
+      @include column(4, $card-width);
+    }
+  }
+
+  @media (min-width: 768px) and (max-width: 1023px) {
+    & {
+      @include column(3, $card-width);
+    }
+  }
+
+  @media (min-width: 488px) and (max-width: 767px) {
+    & {
+      @include column(2, $card-width);
+    }
+  }
+
+  @media (max-width: 487px) {
+    & {
+      @include column(1, $card-width);
+    }
+  }
```

Comment:
Tip: you can write this `@include` simply in `.catalog` without media queries. It will be treated as default setting and will be overwritten by another queries if those are below it

Reply:
> Looks and works fine, good job :) However, I have left some tips in comments, would be great if you applied them

Thank you, it makes sense. In fact, I've relized all of it reviewing your own code.